src/app/migration/src/src/app/modules/workspace/components/draft/draft.component.html

```diff
@@ -0,0 +1,40 @@
+<!-- Loader -->
+<div class="twelve wide column margin-top-20" *ngIf="showLoader">
+  <app-loader></app-loader>
+</div>
+
+<div class="ui grid">
+  <div class="twelve wide column">
+    <div class="ui three stackable cards">
+      <div *ngFor="let content of drafList">
+       <app-content-card [content]="content" (clickEvent)="deleteDraft($event)" ></app-content-card>
+      </div>
+    </div>
+  </div>
+</div>
+<div class="ui grid">
```

Comment:
if no content found, You don't have any draft content... message should be shown